.env

```diff
@@ -25,3 +25,9 @@ EXCLUDE_FROM_LB_WAIT_TIME_IN_SEC=0
 # Note: the "unlimited" option does not work well with `EXCLUDE_FROM_LB_WAIT_TIME_IN_SEC`
 # option, which could potential exclude all nodes from load balancer at the same time.
 MAX_CONCURRENT_UPDATE=1
+
+# Prevent update time window setting
+# Brupop will prevent node updates within time window
+# when you set up time window start and end time, you should use UTC (24-hour time notation).
+PREVENT_UPDATE_TIME_WINDOW_START= 0:0:0
+PREVENT_UPDATE_TIME_WINDOW_END= 0:0:0
```

Comment:
I think I actually prefer using "start" and "end" as the in name. "Since" and "until" are a bit confusing because "since" could _by itself_ indicate a time period.

But whatever @gthao313 thinks is best works for me!

Reply:
Yeah! I would like use `start` and `end`